### ISSUE TITLE
Log file download does not always work

### ISSUE THREAD

**Issue by MichaelRoeder** [hobbit-project/platform#220]
## Problem

When clicking the log file button, the log file is not always downloaded.

When inspecting the network traffic, it looks like the data is requested and received, e.g.,
```
[{"_index":"logstash-2018.02.13","_type":"other","_source":{"image_name":"git.project-hobbit.eu:4567/ernestoj/logmapsystem","@timestamp":"2018-02-13T13:26:57.773Z","container_name":"logmapsystem-a53bc9caaad546bc90d2c57276daf10a","message":"[main] INFO org.hobbit.logmap.LogMapSystemAdapter - LogMap System Adapter closed successfully.","container_id":"714d1114a15d632100a549759faa445be413327ebc4a7930c08d2351979f7c09"},"_id":"AWGPWM_0xaKjLXuxNP8r","sort":[1518528417773],"_score":null},
...
{"_index":"logstash-2018.02.13","_type":"other","_source":{"image_name":"git.project-hobbit.eu:4567/ernestoj/logmapsystem","@timestamp":"2018-02-13T13:26:15.993Z","container_name":"logmapsystem-a53bc9caaad546bc90d2c57276daf10a","message":"[main] INFO org.hobbit.logmap.LogMapSystemAdapter - LogMap initialized.","container_id":"714d1114a15d632100a549759faa445be413327ebc4a7930c08d2351979f7c09"},"_id":"AWGPWCzAxaKjLXuxNP01","sort":[1518528375993],"_score":null}]
```
but an exception occurs in the script:
```
ERROR TypeError: e.text is not a function
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.__tryOrUnsub (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.notifyNext (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
```

This error was reported for the experiment http://master.project-hobbit.eu/experiments/1518528351310

**Comment by MichaelRoeder** [comment]
@Ennosigaeon any progress regarding this problem?

**Comment by earthquakesan** [comment]
It is still an issue on the develop branch:
```
ERROR TypeError: e.text is not a function
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.__tryOrUnsub (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.notifyNext (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
    at t._next (main.f73a8c6c03190195c914.bundle.js:1)
    at t.next (main.f73a8c6c03190195c914.bundle.js:1)
```

**Comment by jsaveta** [comment]
Not able to download log files from Mozilla Firefox.  On the other hand, the files are downloaded with Chrome.

**Comment by Ennosigaeon** [comment]
A working solution for Firefox and Chrome is available in pull request #342. I will not test / support any other browsers.
